lib/sycamore/sycamore/query/operators/count.py

```diff
@@ -1,39 +1,16 @@
-from typing import Any, Dict, Optional
+from typing import Optional
 
 from sycamore.query.operators.logical_operator import LogicalOperator
 
 
 class Count(LogicalOperator):
-    """
-    Logical count operator for determining number of records; takes form of basic or unique count.
-    """
+    """Determines the length of a particular database (number of records).
 
-    def __init__(self, node_id: str, data: Optional[Dict[Any, Any]] = None):
-        super().__init__(node_id, data)
+    Returns a number.
+    """
 
-    @staticmethod
-    def description() -> str:
-        return """
-        **Count**: Determines the length of a particular database (number of records). Parameters are *description*,
-            *input*, *field*, *primaryField*, and *id*. Returns a number.
-        - *description* is a written description of the purpose of this operation in this context
-            and justification of why you chose to use it.
-        - *input* is a list of operation ids that this operation depends on. For this operation,
-            *input* should only contain one id of an operation that returns a database (len(input) == 1).
-        - *field* is an optional parameter. non-primary database field to return a count based on.
-        - *primaryField* is an optional parameter. It is a primary field that represents what a
-              unique entry is considered for the data provided.
-        - *id* is a uniquely assigned integer that serves as an identifier.
-        """
+    field: Optional[str] = None
+    """Non-primary database field to return a count based on."""
```

Comment:
would the generated prompt also say this is optional?

Reply:
Yes, because the type hint that is included as part of the prompt says `typing.Optonal[str]`. At least GPT-4o seems to understand Python syntax when describing the types, but we may need to tweak how this is rendered to the LLM prompt if that does not work.